src/Faker/Provider/en_US/Company.php

```diff
@@ -90,4 +96,15 @@ public function bs()
 
         return join($result, ' ');
     }
+
+    /**
```

Comment:
Can you add a more explicit documentation (like "Employer Identification Number" in phpDoc, with a link to what it is)?

Reply:
Wanted to take a quick second to follow up on this. This particular change was pushed up some time ago, but GitHub doesn't seem to have actually flagged it. Just didn't want this flying under the radar for too long.